test/support/http_unauthorized_error_plug.ex

```diff
@@ -0,0 +1,18 @@
+defmodule NervesHubLink.Support.HTTPUnauthorizedErrorPlug do
+  @moduledoc false
+
+  @behaviour Plug
+
+  import Plug.Conn
+
+  @impl Plug
+  def init(options), do: options
+
+  @impl Plug
+  def call(conn, opts) do
+    send(opts[:report_pid], {:request_error})
```

Comment:
```suggestion
    send(opts[:report_pid], :request_error)
```
Prob doesn't need to be a tuple?

Reply:
true, I ran into an issue where I thought I needed the curly brackets, but I resolved the issue.

thanks!